dash.el

```diff
@@ -2115,7 +2119,7 @@ because we need to support improper list binding."
          (-let* ,new-varlist ,@body)))))
 
 (defmacro -lambda (match-form &rest body)
-  "Return a lambda which destructures its input as MATCH-FORM and executes BODY.
+  "Return a lambda which destructures its input as MATCH-FORM and execute BODY.
```

Comment:
I'm not going to comment on this anymore, but go through it and put back to third person (-s) those where it is gramatically necessary.